src/xarm-commander.cpp

```diff
@@ -0,0 +1,157 @@
+#include <CLI11.hpp>
+#include <xarm/wrapper/xarm_api.h>
+
+#include <string>
+#include <iostream>
+
+int main(int argc, char **argv)
+{
+    XArmAPI *arm;
+    std::string port = "130.82.171.9";
+    int res = 1;
+
+    CLI::App app{"xarm-commander: a command line tool for controlling the xArm7."};
+
+    // print defaults on help
+    app.option_defaults()->always_capture_default();
+
+    // ===== FLAGS =====
+    bool print_mode = false;
+    app.add_flag("-v, --verbose", print_mode, "Verbose mode."); // TODO: multilevel verbose modes + logger
+
+    app.require_subcommand(1); // set max number of subcommands to 1
+
+    // ===== SUBCOMMANDS =====
+    // Subcommand: motion_enable
+    auto motion_enable = app.add_subcommand("motion_enable", "Motion enable");
+
+    bool enable_option = true;
+    motion_enable->add_option("-e, --enable", enable_option, "Enable or disable");
+
+    int servo_option = 8;
+    motion_enable->add_option("-s, --servo", servo_option, "Choose servo (1-8), 8: enable/disable all servo ");
+
+    motion_enable->callback([&]() {
+        arm = new XArmAPI(port);
+        res = arm->motion_enable(enable_option, servo_option);
+
+        if (print_mode)
```

Comment:
Let's put braces even if it's a single line. This concreteness is adopted by Go, and I believe it makes the code safer while doing no harm.